src/main/java/com/cleanroommc/groovyscript/compat/mods/aether/Accessory.java

```diff
@@ -0,0 +1,87 @@
+package com.cleanroommc.groovyscript.compat.mods.aether;
+
+import com.cleanroommc.groovyscript.api.GroovyLog;
+import com.cleanroommc.groovyscript.api.documentation.annotations.*;
+import com.cleanroommc.groovyscript.helper.Alias;
+import com.cleanroommc.groovyscript.helper.recipe.AbstractRecipeBuilder;
+import com.cleanroommc.groovyscript.registry.ForgeRegistryWrapper;
+import com.cleanroommc.groovyscript.registry.ReloadableRegistryManager;
+import com.gildedgames.the_aether.api.accessories.AccessoryType;
+import com.gildedgames.the_aether.api.accessories.AetherAccessory;
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.fml.common.registry.GameRegistry;
+import org.jetbrains.annotations.Nullable;
+
+@RegistryDescription
+public class Accessory extends ForgeRegistryWrapper<AetherAccessory> {
+
+    private static final AccessoryType[] VALUES = AccessoryType.values();
+
+    public Accessory() {
+        super(GameRegistry.findRegistry(AetherAccessory.class), Alias.generateOfClass(AetherAccessory.class));
+    }
+
+    public void add(AetherAccessory accessory) {
+        if (accessory != null) {
+            ReloadableRegistryManager.addRegistryEntry(this.getRegistry(),accessory);
+        }
+    }
+
+    public void add(ItemStack item, String type) {
```

Comment:
should be annotated for documentation via something like `@MethodDescription(type = MethodDescription.Type.ADDITION)`